frontend/src/app/modals/autofill-modal/autofill-modal.component.html

```diff
@@ -0,0 +1,78 @@
+<ngbx-modal #modal [back]="['../']" size="lg">
+  <ng-container modal-title>
+    <h4 class="modal-title" id="modal-title">Autofill</h4>
+  </ng-container>
+  <ng-container modal-body>
+    <form [formGroup]="modalForm">
+      <div class="row">
+        <div class="col-md">
+          <label for="dates">Select multiple dates</label>
+          <ngb-datepicker
+            class="form-control"
+            id="dates"
+            (dateSelect)="onDateSelect($event)"
+            [dayTemplate]="dayTemplate"
+            [displayMonths]="1"
+          ></ngb-datepicker>
+        </div>
+        <div class="col-md">
+          <div class="mb-3 ">
+            <label for="startTime">Start time</label>
+            <input class="form-control"
+                   id="startTime"
+                   formControlName="startTime"
+                   type="time"
+            >
+          </div>
+          <div class="mb-3 ">
+            <label for="duration">Duration</label>
+            <input class="form-control"
+                   id="duration"
+                   formControlName="duration"
+                   type="time"
+            >
+          </div>
+          <div class="mb-3 ">
+            <label for="pause">Break between Events</label>
+            <input class="form-control"
+                   id="pause"
+                   formControlName="pause"
+                   type="time"
+            >
+          </div>
+          <div class="mb-3">
+            <label for="repeat">Repeat x times</label>
+            <input class="form-control"
+                   id="repeat"
+                   formControlName="repeat"
+                   type="number"
+                   autocomplete="off"
+                   min="1"
+            >
+          </div>
+        </div>
+      </div>
+    </form>
+  </ng-container>
+  <ng-container modal-footer>
+    <div class="text-muted bi-info-circle">
+      End time: {{ endTime }}
+    </div>
```

Comment:
I think it would be useful to add a hint text in addition to the info-circle, which explains end time in more detail.
"The end time is calculated based on the start time you specify, the duration of the events, the pauses between events, and the number of repetitions."

Reply:
Done!